js/ui/drop_down_editor/ui.drop_down_editor.js

```diff
@@ -617,7 +617,30 @@ const DropDownEditor = TextBox.inherit({
     },
 
     _popupPositionedHandler: function(e) {
-        e.position && this._popup.$overlayContent().toggleClass(DROP_DOWN_EDITOR_OVERLAY_FLIPPED, e.position.v.flip);
+        const $popupOverlayContent = this._popup.$overlayContent();
+        const isOverlayFlipped = e.position.v.flip;
+        const labelMode = this.option('labelMode');
+        const stylingMode = this.option('stylingMode');
+
+        const isNeedMoved = (labelMode, stylingMode) => (
```

Comment:
Why you decided that we need to do anything for underlined mode?

Reply:
https://codepen.io/marker-dao/pen/WNzVVZG